test/cloud-metadata/index.test.js

```diff
@@ -254,28 +254,6 @@ tape('cloud metadata: aws empty data', function (t) {
   });
 });
 
-tape('cloud metadata: gcp empty data', function (t) {
```

Comment:
Reviewer note: I've dropped a couple unnecessarily defensive test cases, IMHO. See the note above about the statusCode==200 guard.